internal/kibana/alerting_test.go

```diff
@@ -18,6 +18,8 @@ import (
 func TestAccResourceAlertingRule(t *testing.T) {
 	minSupportedVersion := version.Must(version.NewSemver("7.14.0"))
 
+	t.Setenv("KIBANA_API_KEY", "")
```

Comment:
Do we need to to do this?

Reply:
Makes sense, just so I'm clear though, it's not currently possible to use API key auth with the connectors resources? 

> The issue is that connectors did not support ApiKey scheme, only Bearer

I'm assuming you mean `Authoization: Bearer <key>` works, but `Authorization: ApiKey <key>` does not? Looking at the test run does that mean the ApiKey support is added in 8.11 (is there an issue here)?

I think we should add a note in the docs for the impacted resources if the above is correct.

Reply:
Apologies, I linked to the wrong test output: https://github.com/elastic/terraform-provider-elasticstack/actions/runs/6930946037/job/18851625387

Reply:
It looks like as of 8.11 it still does not support `ApiKey` schemas in authorization headers:

https://www.elastic.co/guide/en/kibana/current/enable-rule-api.html

I'll add a note on connectors in the docs 👍🏻